src/styles/main.scss

```diff
@@ -1,3 +1,204 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+}
+
+a,
+body,
+h2,
+img {
   margin: 0;
 }
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  max-width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 25px 30px;
+
+  background: #fff;
+  &__imac {
+    width: 160px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+
+    margin-bottom: 2px;
+
+    color: #060b35;
+    align-self: center;
+
+    line-height: 18px;
+  }
+
+  &__code {
+    font-weight: 400;
+
+    color: #616070;
+
+    align-self: flex-start;
+
+    margin-bottom: 16px;
+
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__feed {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__label {
+    font-weight: 400;
+    font-size: 12px;
+    color: #616070;
+    line-height: 18px;
+  }
+
+  &__number {
+    font-weight: 700;
+    font-size: 16px;
+    list-style: 18px;
+  }
+}
+
+.button {
+  width: 168px;
```

Comment:
Same thing. If you have styles for class or its elements - move it to separate file and use imports.